run_max_ent.py

```diff
@@ -0,0 +1,103 @@
+"""Script running the MaxEnt IRL
+algorithm on a trees data set.
+Usage:
+    python scripts/run_max_ent.py data=$PWD/path_to_trees.csv n_action=5
+Note:
+    The data path must be absolute not relative.
+"""
+import dataclasses
+from locale import normalize
+import logging
+from typing import Any, Dict, Optional
+
+import anytree
+from matplotlib.pyplot import get
+import pandas as pd
+import numpy as np
+
+import phirl.api as ph
+import phirl.hydra_utils as hy
+from phirl.maxent import *
+from irl_maxent import optimizer as O
```

Comment:
I'd suggest just `from irl_maxent import optimizer` – `O` is easy to confuse with `0`, plus this import syntax (importing as a single uppercase letter) is quite rare. (The only exception to this rule is `pytorch.nn.functional`, but I really don't know why it's that).